New Docusaurus update 3.2.1 causing some build issues

Currently with:

- Docusaurus 3.2.1
- Node 20.12.0
- Yarn 4.1.1

I am getting errors during `yarn build` or `yarn start` like the following:

```
E:\GitHubRepos\openrefine.org\docs>yarn start
[INFO] Starting the development server...

[ERROR] Error: Docusaurus could not load module at path "E:\GitHubRepos\openrefine.org\src\plugins\docusaurus-versionsjson-plugin\index.js"
Cause: Your application tried to access fs-extra, but it isn't declared in your dependencies; this makes the require call ambiguous and unsound.

Required package: fs-extra
Required by: E:\GitHubRepos\openrefine.org\src\plugins\docusaurus-versionsjson-plugin\

Require stack:
- E:\GitHubRepos\openrefine.org\src\plugins\docusaurus-versionsjson-plugin\index.js
    at loadFreshModule (E:\GitHubRepos\openrefine.org\.yarn\__virtual__\@docusaurus-utils-virtual-daa5b7b8bd\0\cache\@docusaurus-utils-npm-3.2.1-cfade3be35-0ed193119b.zip\node_modules\@docusaurus\utils\lib\moduleUtils.js:36:15)

etc.
```


My `package.json` currently looks like this:

```json
{
  "name": "OpenRefine-Documentation",
  "version": "1.0.0",
  "description": "Documentation for OpenRefine",
  "license": "BSD-3-Clause",
  "author": "OpenRefine",
  "private": true,
  "scripts": {
    "start": "docusaurus start",
    "build": "docusaurus build",
    "swizzle": "docusaurus swizzle",
    "deploy": "docusaurus deploy",
    "netlify:build:production": "yarn build",
    "netlify:build:deployPreview": "yarn build"
  },
  "dependencies": {
    "@docusaurus/core": "^3.2.1",
    "@docusaurus/preset-classic": "^3.2.1",
    "@docusaurus/theme-mermaid": "^3.2.1",
    "clsx": "^1.2.1",
    "create-docusaurus": "^2.1.0",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-lite-youtube-embed": "^2.3.1",
    "ua-parser-js": "^1.0.37"
  },
  "packageManager": "yarn@4.1.1"
}
```

I wouldn't think I have to manually add `fs-extra` as a dependency here since Yarn / Docusaurus should do that for me with `yarn up` ?  What's going on...hmm.

--- a/README.md
+++ b/README.md
@@ -1,19 +1,25 @@
 OpenRefine.org website
 ======================
 
-We use [Docusaurus 2](https://v2.docusaurus.io/) for our website.
+We use [Docusaurus](https://docusaurus.io/) for our website.
 
-If you want to run it locally, you can follow the instructions below, otherwise you should be able to edit most pages directly with a Markdown editor. Opening a pull request for your changes will generate a preview that you can browse to check the
-rendering.
+If you want to run it locally, you can follow the instructions below, otherwise you should be able to edit most pages directly with a Markdown editor.
+Opening a pull request for your changes will generate a preview that you can browse to check the rendering.
 
 ### Requirements
 
-Assuming you have [Node.js (LTS recommended)](https://nodejs.org/en/download/) installed (which includes npm), you can install Docusaurus with:
+Assuming you have [Node.js (LTS recommended)](https://nodejs.org/en/download/) installed (which now includes corepack), you can install Docusaurus with:
 
-You will need to install [Yarn](https://yarnpkg.com/getting-started/install) before you can build the site.
+Enable Corepack, if it isn't already; this will add the [Yarn](https://yarnpkg.com/getting-started/install) binary to your PATH:
 
 ```sh
-npm install -g yarn
+corepack enable
+```
+
+Update Yarn to the latest version:
+
+```sh
+yarn set version stable
 ```
 
 ### Installation
@@ -45,10 +51,10 @@ TypeError: Cannot read property 'latest' of undefined
 error Command failed with exit code 1.
 ```
 
-then it is likely that you will need to first run
+then it is likely that you will need to first upgrade dependencies across all workspaces at the same time
 
 ```sh
-yarn upgrade
+yarn up
 ```
 
 which will update all dependencies and store them in the `yarn.lock` file to also be committed.

